splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceUnitTest.java

```diff
@@ -1195,6 +1195,21 @@ protected void checkBooleanExpression(String input, boolean output, TestConnecti
             rs.next();
             Assert.assertEquals(output, rs.getBoolean(1));
         }
+    }
+
+    protected void checkExpressionType(String input, String expectedType, TestConnection conn) throws SQLException {
+        String sql = format("select typeof(%s)", input);
+        try(ResultSet rs = conn.query(sql)) {
+            rs.next();
+            Assert.assertEquals(expectedType, rs.getString(1));
+        }
```

Comment:
This logic is available as `firstRowContainsQuery()` in the same file. But it's just a minor issue and should not prevent the change from being merged.